payload/builder_test.go

```diff
@@ -146,6 +147,45 @@ func TestCategory(t *testing.T) {
 	assert.Equal(t, `{"aps":{"category":"NEW_MESSAGE_CATEGORY"}}`, string(b))
 }
 
+func TestContentState(t *testing.T) {
+	payload := NewPayload().SetContentState(map[string]interface{}{"my_int": 13, "my_string": "foo"})
+	b, _ := json.Marshal(payload)
+	assert.Equal(t, `{"aps":{"content-state":{"my_int":13,"my_string":"foo"}}}`, string(b))
+}
+
+func TestDismissalDate(t *testing.T) {
+	timestamp := time.Date(2023, 1, 27, 12, 14, 00, 00, time.UTC).Unix()
+	payload := NewPayload().SetDismissalDate(timestamp)
+	b, _ := json.Marshal(payload)
+	assert.Equal(t, `{"aps":{"dismissal-date":1674821640}}`, string(b))
+}
+
+func TestStaleDate(t *testing.T) {
+	timestamp := time.Date(2023, 1, 27, 12, 14, 00, 00, time.UTC).Unix()
+	payload := NewPayload().SetStaleDate(timestamp)
+	b, _ := json.Marshal(payload)
+	assert.Equal(t, `{"aps":{"stale-date":1674821640}}`, string(b))
```

Comment:
What's 1674821640? should it be in a constant to be more readable?

Reply:
it's the unix timestamp of `time.Date(2023, 1, 27, 12, 14, 00, 00, time.UTC)` - I think it's nice to have the clean string and not need to import fmt